### ISSUE TITLE
Stage button no longer works

### ISSUE THREAD

**Issue by matkatmusic** [ForkIssues/Tracker#993]
The Stage button and Unstage button no longer refresh the staged changes/unstaged changes view in v1.0.92.
the Stage All Files button works still.

Steps to re-produce:

1. Select a file that needs staging.
2. click the stage button
3. no files move to the 'staged' area.
4. Click on Commits view, then go back to Changes view. 
5. the file is now staged. 
6. Select that staged file.
7. click the un-stage button.  the file is not unstaged.
8. Click on Commits view, then go back to Changes view. 
9. the file is now unstaged.

**Comment by DanPristupov** [comment]
Most likely you use an old git version: https://github.com/ForkIssues/Tracker/issues/968.

Update git or switch Fork to the internal one.

**Comment by matkatmusic** [comment]
This didn't start happening until I updated to 1.0.92.  it was fine in
v1.0.91.   it's like the [stage] button doesn't trigger the animation or
refresh the staged/unstaged list boxes..

**Comment by DanPristupov** [comment]
> This didn't start happening until I updated to 1.0.92.  it was fine in v1.0.91. 

Yes. In 1.0.92 Fork started using new parameters to prevent repository lock on getting list of changed files.

**Comment by matkatmusic** [comment]
Which version of git would you recommend I change to?  the one provided by
Fork?  The one in `/user/local/bin/git` instead of the one from Apple at
`user/bin/git` ?

**Comment by DanPristupov** [comment]
`/user/local/bin/git` is probably the one installed with brew. You can use it, if it's relatively fresh. The one inside Fork should be the most stable version.

Just ensure that fetch/push still works for you. You may need to configure it again.

**Comment by gernotstarke** [comment]
oops - after switching to the fork-provided git (2.23.0), github credentials cannot be retrieved from OSX keychain:

```
Pushing to https://github.com/xxx.git
git: 'credential-osxkeychain' is not a git command. See 'git --help'.
dyld: Library not loaded: @rpath/libswiftCore.dylib
  Referenced from: /Applications/Fork.app/Contents/Resources/fork_askpasswd
  Reason: image not found
error: /Applications/Fork.app/Contents/Resources/fork_askpasswd died of signal 6
error: unable to read askpass response from '/Applications/Fork.app/Contents/Resources/fork_askpasswd'
fatal: could not read Username for 'https://github.com': Device not configured
```

(was trying to adress the problem, that the "stage" button does not work properly any more)

**Comment by DanPristupov** [comment]
@gernotstarke  https://github.com/ForkIssues/Tracker/issues/611

Swift 5 command line programs require the Swift 5 libraries. These libraries are included in the OS starting with macOS Mojave 10.14.4. When running on earlier versions of macOS you must install the following package manually.
https://support.apple.com/kb/DL1998?locale=en_US

**Comment by gernotstarke** [comment]
thx Dan!
